src/main/java/com/dezorganizacja/bridge/presenter/MainPresenter.java

```diff
@@ -2,8 +2,11 @@
 
 import com.dezorganizacja.bridge.domain.Message;
 
-public class MainPresenter {
+import java.util.Observable;
+
+public class MainPresenter extends Observable {
     private final Message message;
+    private String state;
```

Comment:
Nie twierdzę, że to musi być bijekcja. Ot funkcja ze stanu gry w scenę. :stuck_out_tongue: